tests/nat/eval/runners/red_teaming_runner/test_red_teaming_runner.py

```
@@ -0,0 +1,203 @@
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import cast
+
+import pytest
+
+import nat.middleware.register  # noqa: F401  # Import register module to trigger registration
```

Ruff (0.14.8)

Unused `noqa` directive (non-enabled: `F401`)

Remove unused `noqa` directive

(RUF100)